app/app_test.go

```diff
@@ -69,45 +88,51 @@ func (at *appTest) reset() {
 	require.True(at.t, resabci.IsOK(), resabci)
 }
 
+func getBalance(pk crypto.PubKey, state types.KVStore) (types.Coins, error) {
+	return getAddr(pk.Address(), state)
+}
+
+func getAddr(addr []byte, state types.KVStore) (types.Coins, error) {
+	actor := stack.SigPerm(addr)
+	acct, err := coin.NewAccountant("").GetAccount(state, actor)
```

Comment:
`acc` confused me.  seems like accumulator.  acct is a universal abreviation for account (I see "Acct #"  in bills, never "Acc #")

Basically, I think there were some bad namings historically, and I do not want to carry them all on.  Let's use the name that is better to a newcomer, and standardize them.